packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -53,6 +54,13 @@ export interface IRatingProps extends React.HTMLProps<HTMLElement> {
    * Optional id of label describing this instance of Rating
    */
   ariaLabelId?: string;
+
+  /**
+   * Optional setting to represent aggregate rating values on rating control.
+   * If set, rating control will be readonly rating control with partial star support.
+   * Also if set, this overrides the disabled prop
+   */
+  aggregate?: boolean;
```

Comment:
I disagree with "this keeps the control simple to use and non confusing". I still think trying to squeeze two different use cases into a single control doesn't make sense and makes it confusing to use.